loadtest/user/userentity/actions.go

```diff
@@ -148,6 +148,23 @@ func (ue *UserEntity) CreatePost(post *model.Post) (string, error) {
 	return post.Id, err
 }
 
+func (ue *UserEntity) PatchPost(postId string, patch *model.PostPatch) (string, error) {
+	post, resp := ue.client.PatchPost(postId, patch)
+	if resp.Error != nil {
+		return "", resp.Error
+	}
+
+	if err := ue.store.DeletePost(postId); err != nil {
+		return "", err
+	}
+
+	if err := ue.store.SetPost(post); err != nil {
+		return "", err
+	}
```

Comment:
Do we need to delete here? I think `Set` overwrites whatever is there.

Reply:
Yes you are right, I was not so sure if they have the same `post.Id`. But after some tests, I observed that the id is not changing. We can remove `ue.store.DeletePost(postId)`.